modules/efk-logging-deploy-subscription.adoc

```diff
@@ -27,9 +27,152 @@ The initial set of {product-title} nodes might not be large enough to support th
 {product-title} cluster to run with the recommended or higher memory. Each Elasticsearch node can operate with a lower
 memory setting though this is not recommended for production deployments.
 
-. Create a project for cluster logging. You must create the project with the CLI:
 
-.. Create a YAML file with the following:
+.Procedure
+
+. Install the Elasticsearch Operator using the CLI to ensure the necessary values are set:
+
+.. Create a namespace for the Elasticsearch operator (for example, `eo-project.yaml`):
++
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true"
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+
+.. Create the Namespace object:
++
+----
+$oc create -f eo-project.yaml
+----
+
+.. Create a Subscription object YAML file (for example, `eo-sub.yaml`) to
```

Comment:
I think you have to create the CatalogSourceConfig before the Subscription - at least, that's the way logging CI works.